### DISCUSSION TITLE
Is latchset/mod_auth_mellon dead?

### DISCUSSION THREAD

**Discussion by manu0401** [latchset/mod_auth_mellon#126] [Q&A]
Hello

No commit for years, pull requests and issues pending. Is this repository still alive? 

Was mod_auth_mellon developement moved somewhere else?

**Comment by simo5** [comment]
The last commit is from January and there is a PR opened last month.

The project is in maintenance mode, there isn't much to develop at this stage as it does what it was built for, but it is still alive.
